examples/dynamo_persistence.py

```diff
@@ -59,15 +59,15 @@ def main():
         }
     )
     runner = Sync(executor=executor)
-    tasks = set()
     TaskConfig = mesos_executor.TASK_CONFIG_INTERFACE
-    for _ in range(1, 2):
-        task_config = TaskConfig(
-            image='ubuntu:14.04', cmd='/bin/sleep 2'
-        )
-        tasks.add(task_config.task_id)
-        runner.run(task_config)
-        print(executor.status(task_config.task_id))
+
+    for _ in range(2):
+        task_config = TaskConfig(image='ubuntu:14.04', cmd='/bin/true')
+        task_id = runner.new_task_id()
```

Comment:
I think the case to pass in a task_id instead of relying on the runner to generate one is when a task_id is needed later, for example, to check status or terminate its execution. I would recommend not using new_task_id in examples unless it is used later.